registry/remote/credentials/testdata/whitespace.json

```diff
@@ -0,0 +1 @@
+
```

Comment:
nit: We should put some white spaces here if we allow it

Reply:
There is whitespace in there